redbeat/schedulers.py

```diff
@@ -191,11 +191,13 @@ def __init__(self, app, **kwargs):
     def setup_schedule(self):
         # cleanup old static entries
         client = redis(self.app)
-        previous = client.smembers(self.app.conf.REDBEAT_STATICS_KEY)
-        current = set(self.app.conf.CELERYBEAT_SCHEDULE.keys())
-        removed = previous - current
+        previous = set(key.decode('utf-8')
```

Comment:
In fact, redis-py supports the `decode_reponses` arguments to the `StrictRedis`, which makes it decode all responses from Redis to unicode. My first approach for #33 was to set this to `True` and it generally works, but the tests use `fakeredis`, which does not support `decode_responses` (the [PR for adding it](https://github.com/jamesls/fakeredis/pull/116) doesn't seem likely to be merged anytime soon).

Also, I find it much clearer to explicitly to decode the Redis reponses in each case where it is necessary. Since redbeat has control over what goes into the Redis keys it reads, there should also be no issue with not knowing which encoding to use. Lastly, it makes is more robust in case somebody replaces the default `StrictRedis` instance redbeat uses.